google/cloud/spanner_v1/data_types.py

```diff
@@ -71,3 +75,109 @@ def serialize(self):
             return json.dumps(self._array_value, sort_keys=True, separators=(",", ":"))
 
         return json.dumps(self, sort_keys=True, separators=(",", ":"))
+
+
+def __proto_message(bytes_val, proto_message_object):
```

Comment:
Why are we using double underscores here? My understanding is that the intent was for it to be a private fxn and hence should be a single underscore.
Reference: [link](https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-single-and-double-underscore-before-an-object-name)

Reply:
This is a bit confusing as different sources say it differently.
Source: https://www.geeksforgeeks.org/private-methods-in-python/
Text: to define a private method prefix the member name with the double underscore “__”.

However, I will modify it to contain single underscore to follow the standard convention used here.

Reply:
Yes, thankyou